docs/content/preview/yugabyte-cloud/cloud-security/cloud-security-features.md

```diff
@@ -11,7 +11,7 @@ weight: 10
 type: docs
 ---
 
-YugabyteDB Managed is a fully managed YugabyteDB-as-a-Service that allows you to run YugabyteDB clusters on public cloud providers such as Google Cloud Platform (GCP) and Amazon Web Services (AWS), with more public cloud provider options coming soon. YugabyteDB Managed runs on top of [YugabyteDB Anywhere](../../../yugabyte-platform/). It is responsible for creating and managing customer YugabyteDB clusters deployed on cloud provider infrastructure.
+YugabyteDB Managed is a fully managed YugabyteDB-as-a-Service that allows you to run YugabyteDB clusters on public cloud providers such as Google Cloud Platform (GCP), Microsoft Azure, and Amazon Web Services (AWS). YugabyteDB Managed runs on top of [YugabyteDB Anywhere](../../../yugabyte-platform/). It is responsible for creating and managing customer YugabyteDB clusters deployed on cloud provider infrastructure.
```

Comment:
Suggest the ordering to be: Amazon Web Services (AWS), Microsoft Azure, and Google Cloud Platform (GCP)